pkgs/development/python-modules/zeversolarlocal/default.nix

```diff
@@ -21,6 +22,19 @@ buildPythonPackage rec {
     sha256 = "ExZy5k5RE7k+D0lGmuIkGWrWQ+m24K2oqbUEg4BAVuY=";
   };
 
+  # patch below won't apply due to different line endings
+  prePatch = ''
```

Comment:
```suggestion
  postPatch = ''
```

Reply:
This needs to run before `patchPhase` where `patches` are applied